packages/ui/common/src/lib/environments/environment.ts

```diff
@@ -1,6 +1,6 @@
 export const environment = {
   production: false,
-  apiUrl: 'http://localhost:3000/v1',
+  apiUrl: 'https://legendary-garbanzo-q9995r99p4pfx6px-3000.app.github.dev/v1',
```

Comment:
revert?

Reply:
Yes, What I need to do, could not understand,

I followed the tutorial of pieces development. 

Let me know, what I need to change in them, you can edit from your side, if possible

Reply:
If I'm not mistaken, this file should not be modified when adding a piece. Just remove your changes.